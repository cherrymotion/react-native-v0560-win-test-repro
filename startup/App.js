import React from 'react';
import { View, Text } from 'react-native';

class App extends React.Component {
    async componentWillMount() {
        // console.disableYellowBox = true;
    }

    render() {
        return (
            <View>
                <Text>{'Hello 0.56.0'}</Text>
            </View>
        );
    }
}

export default App;
