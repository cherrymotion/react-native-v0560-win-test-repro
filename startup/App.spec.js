import App from './App';

describe('App', () => {
    it('Will render', () => {
        expect(App).toMatchSnapshot();
    });
});
