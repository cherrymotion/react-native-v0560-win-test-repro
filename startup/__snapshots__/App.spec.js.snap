// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Will render 1`] = `[Function]`;
